pkg/reconciler/inmemorychannel/dispatcher/inmemorychannel.go

```diff
@@ -159,3 +149,19 @@ func (r *Reconciler) newConfigForInMemoryChannel(imc *v1.InMemoryChannel) (*mult
 		},
 	}, nil
 }
+
+func (r *Reconciler) deleteFunc(obj interface{}) {
+	if acc, err := kmeta.DeletionHandlingAccessor(obj); err == nil {
```

Comment:
Generally Go style prefers inverting these and returning early to avoid the deep indentation.